components/common/comment_icon.jsx

```diff
@@ -27,6 +28,10 @@ export default class CommentIcon extends React.PureComponent {
         location: Locations.CENTER,
     };
 
+    getTooltipTriggers = () => {
```

Comment:
Were you planning on using this? Perhaps for line 64? Or forgot to remove? It's not being used anymore that I can see. :)